doc/plutus/explanations/ledger.rst

```diff
@@ -0,0 +1,97 @@
+.. _what_is_a_ledger:
+
+What is a ledger?
+=================
+
+The :ref:`Plutus Platform<what_is_the_plutus_platform>` is designed to work with distributed ledgers (henceforth simply “ledgers”).
+Ledgers are typically *implemented* with a blockchain, such as Cardano.
+However, much of the time when we are talking about ledgers we don't care about the underlying implementation, and so we will just talk about the ledger itself.
+
+.. note::
+    This is not always true: applications do need to care about details of how the underlying blockchian work, because that affects behaviour such as settlement time and rollback policies.
```

Comment:
```suggestion
    This is not always true: applications do need to care about details of how the underlying blockchain works, because that affects behaviour such as settlement time and rollback policies.
```